sdks/java/core/src/main/java/org/apache/beam/sdk/io/BigQueryIO.java

```diff
@@ -2124,7 +2125,8 @@ private void flushRows(Bigquery client, TableReference tableReference,
         List<TableRow> tableRows, List<String> uniqueIds) {
       if (!tableRows.isEmpty()) {
         try {
-          BigQueryTableInserter inserter = new BigQueryTableInserter(client);
+          PipelineOptions options = PipelineOptionsFactory.create();
```

Comment:
Please pass in the pipelineoptions from the finishBundle call to the flushRows method